halibot/halibot.py

```diff
@@ -130,10 +136,7 @@ def _instantiate_modules(self):
 			self.add_module_instance(k, obj(self, conf))
```

Comment:
Is there a reason for there to be a separate `add_module_instance` and `add_agent_instance` now? The only thing different is the log string.

Merging these would mean we can merge `_instantiate_agents` and `_instantiate_modules` too.